lips/lip-1.md

```diff
@@ -385,6 +389,23 @@ Represents a national ID.
 }
 ```
 
+### StatusObject
+
+| Field 	    | Type 	| Required? 	| Description 	|
+|-------	    |------	|-----------	|-------------	|
+| status | str enum | Y | Status of the payment from the perspective of this actor. This field can only be set by the respective sender/receiver VASP and represents the status on the sender/receiver VASP side. This field is mandatory by this respective actor (either sender or receiver side) and mutable. Valid values are specified in [ StatusEnum ](#statusenum)  |
+| abort_code    | str (enum) | N    | In the case of an `abort` status, this field may be used to describe the reason for the abort. Represents the error code of the corresponding error |
+| abort_message         | str      | N             | Additional details about this error.  To be used only when `code` is populated |
+
+<details>
+<summary> StatusObject example </summary>
+<pre>
+{
+    "status": "needs_kyc_data",
+}
+</pre>
+</details>
+
```

Comment:
interestingly, in the travel_rule_data_exchange.md in our private repo, this "preview" dominates but here this is the only(?) occurrence, code block is used elsewhere